services/records/records/schemas.go

```diff
@@ -0,0 +1,124 @@
+package records
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+var validRecordConstraintTypes = []string{"UNKNOWN", "PASSED", "FAILED"}
+
+var recordSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"record_template_id": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"name": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"record_state": {
```

Comment:
```suggestion
	"state": {
```

Reply:
Still present

Reply:
Oups again, I just fix it in RecordTemplate. It's also fixed in Record now.